sycl/test/lit.cfg.py

```diff
@@ -85,7 +85,8 @@
         print("Adding path to opencl-aot tool to PATH")
         os.environ['PATH'] = os.path.pathsep.join((os.getenv('PATH'), config.sycl_tools_dir))
 
-backend=lit_config.params.get('SYCL_BE', "PI_OPENCL")
+backend = lit_config.params.get('SYCL_BE', "PI_OPENCL")
+config.environment['SYCL_BE'] = backend
```

Comment:
This is a great idea!
Please remove [`gpu_run_on_linux_substitute += " SYCL_BE=PI_CUDA "`](https://github.com/intel/llvm/pull/1288/files#diff-a9d09cdd3aae7d1d404b131f4ac6482dR159) below.

I have a similar but less elegant PR in the works that I will adapt to your change.